src/modules/permission/permission.routes.js

```diff
@@ -0,0 +1,13 @@
+const { createPermission } = require("./permission.controller.js");
+const validate = require("../core/middlewares/validator.middleware");
+const { permissionSchema } = require("./permission.schema");
+const PermissionStrategy = require("./permission.authentication.middleware");
+
+module.exports = (app) => {
+  app.post(
+    "/api/permissions/create",
```

Comment:
vai route er ta /api/permissions diyen. API endpoint get, create. update, delete sob gula tai same hobe.

Reply:
Ok, vai I will Update it in my next PR.